packages/name/src/utilities/getGraphemes.ts

```diff
@@ -0,0 +1,21 @@
+type Locale = 'th' | 'ko' | 'ja';
+
+export function getGraphemes({
+  text,
+  locale,
+}: {
+  text?: string;
+  locale: Locale;
+}): string[] {
+  // returns undefined when Intl.Segmenter does not exist in our JS environment (such as in Firefox)
+  if (!text || !Intl.Segmenter) {
+    return undefined;
```

Comment:
Either the return type should specify `string[] | undefined` or this should be changed to return an empty array.